github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits-0.26.yaml

```diff
@@ -454,6 +454,55 @@ presubmits:
           path: /dev/vfio/
           type: Directory
         name: vfio
+  - name: pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6
+    skip_branches:
+    - release-0.26
```

Comment:
doesn't it mean that it will skip release-0.26 branch?

Reply:
Bad copy paste s/skip_branches/branches/